.github/stale.yml

```diff
@@ -0,0 +1,51 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - popular
+  - pinned
+  - security
+  - "[Status] Maybe Later"
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
+
+# Label to use when marking as stale
+staleLabel: stale-bot
+
+# Comment to post when removing the stale label.
+# unmarkComment: >
```

Comment:
Is the idea that it wouldn't comment when removing the label? I think that sounds sensible.

Reply:
Yeah, i think with this config it will say nothing when the stale label is removed. The issue/ pr is not being opened or closed at this point, the bot is just being swatted away.

Should we add a comment for this? Was worried about noise.

Reply:
A code comment or an issue comment? I don't think a comment on the issue is needed here. I doubt a code comment is either, really, since pretty much no one is going to be looking at this.

Reply:
Cool.